core/bin/system-constants-generator/src/main.rs

```diff
@@ -56,23 +60,22 @@ pub fn generate_l1_contracts_system_config(gas_constants: &IntrinsicSystemGasCon
         priority_tx_max_pubdata: (L1_TX_DECREASE * (MAX_PUBDATA_PER_BLOCK as f64)) as u32,
         fair_l2_gas_price: FAIR_L2_GAS_PRICE_ON_L1_CONTRACT,
         l1_gas_per_pubdata_byte: L1_GAS_PER_PUBDATA_BYTE,
-        block_overhead_l2_gas: BLOCK_OVERHEAD_GAS,
+        block_overhead_l2_gas: 0,
```

Comment:
Just of simplification, since L2 gas overhead <<< L1 gas overhead, I decided to remove it altogether to keep the calculations simple. I'll probably just remove the constant to not confuse anyone